### PR TITLE
Add LiveSession / Analytics

### PR DESCRIPTION
**Pull request by kamildrazkiewicz**
Add LiveSession to Wappalyzer.

### CODE REVIEW

File: src/technologies/l.json [hunk 1]
```diff
@@ -1647,5 +1647,22 @@
     "implies": "YouTube",
     "oss": true,
     "website": "https://github.com/paulirish/lite-youtube-embed"
+  },
+  "LiveSession": {
+    "cats": [
+      10
+    ],
+    "description": "LiveSession helps increase conversion rates using session replays, and event-based product analytics.",
+    "icon": "LiveSession.svg",
```
Comment (nurbek91) [inline src/technologies/l.json:1656]: 32x32 with correct proportion 
[LiveSession.svg.zip](https://github.com/wappalyzer/wappalyzer/files/8899716/LiveSession.svg.zip)

**Review by nurbek91 — CHANGES_REQUESTED** [review]
Hey mate thanks for PR !!
Examples: 
https://www.sportgps.com.au/
https://codiciean.it/
https://the-dots.com/

File: src/technologies/l.json (hunk 1 above)
Comment (kamildrazkiewicz) [reply src/technologies/l.json:1656]: fixed

**Comment by IZ6RND** [comment]
> Hey mate thanks for PR !! Examples: https://www.sportgps.com.au/  ************* https://the-dots.com/

Please remove that URL posted. Thank You!

**Comment by IZ6RND** [comment]
> Hey mate thanks for PR !! Examples: ***********************

Jak najszybciej !

**Pull request merged**